### PR TITLE
[libnick] Add libnick V2024.1.3

### PR DESCRIPTION
**Pull request by nlogozzo**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines. ([NickvisionApps/libnick](https://github.com/NickvisionApps/libnick))
- [x] Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html)
- [x] The versioning scheme in `vcpkg.json` matches what upstream says.
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
- [x] The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/libaura/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,29 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
```
Comment (dg0yt) [inline ports/libaura/portfile.cmake:1]: If this is a windows (dllexport) limitation, please guard with `if(VCPKG_TARGET_IS_WINDOWS)`.

File: ports/libaura/vcpkg.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "maintainers": "Nicholas Logozzo nlogozzo225@gmail.com",
+  "description": "A cross-platform base for native Nickvision applications.",
+  "homepage": "https://github.com/NickvisionApps/libaura",
+  "documentation": "https://github.com/NickvisionApps/libaura/tree/main/docs",
+  "license": "GPL-3.0",
+  "supports": "(windows & x64) | (linux & x64)",
+  "dependencies": [
+    "boost-locale",
+    "curl",
+    "gettext",
```
Comment (dg0yt) [inline ports/libaura/vcpkg.json:13]: If this for tools:
```suggestion
    {
      "name": "gettext",
      "host": true
    },
```

File: ports/libaura/vcpkg.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "gettext-libintl",
+    "glib",
+    "gtest",
+    "jsoncpp",
+    {
+      "name": "libsecret",
+      "platform": "linux & x64"
```
Comment (dg0yt) [inline ports/libaura/vcpkg.json:20]: Is x64 necessary here?

File: ports/libaura/vcpkg.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    {
+      "name": "libuuid",
+      "platform": "linux & x64"
```
Comment (dg0yt) [inline ports/libaura/vcpkg.json:24]: Is x64 necessary here?

**Review by dg0yt — CHANGES_REQUESTED** [review]
> For a debug build, depends on https://github.com/microsoft/vcpkg/pull/36055 as CI will run indefinitely without said fix.

No dependency. vcpkg CI has no problem gettext-libintl. And very few users seem to share that issue.

**Comment by nlogozzo** [comment]
Ok, good to go now 😆...sorry for allt the commits....feel free to just squash and merge

File: ports/libaura/usage [hunk 5]
```diff
@@ -0,0 +1,4 @@
+libaura provides CMake targets:
+
+find_package(libaura CONFIG REQUIRED)
+target_link_libraries(main PRIVATE libaura::libaura)
```
Comment (dg0yt) [inline ports/libaura/usage:4]: Missing line break at EOF.

File: ports/libaura/vcpkg.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "libaura",
```
Comment (dg0yt) [inline ports/libaura/vcpkg.json:2]: On repology.org, "libaura" is already take by another package from one repository:
https://repology.org/project/libaura/versions

File: ports/libaura/vcpkg.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  "license": "GPL-3.0",
+  "supports": "(windows & x64) | (linux & x64)",
+  "dependencies": [
+    "boost-locale",
+    "curl",
+    {
+      "name": "gettext",
+      "host": true
+    },
+    "gettext-libintl",
```
Comment (dg0yt) [inline ports/libaura/vcpkg.json:17]: Does it really need (host) gettext (without tools), in addition to libintl?
I don't see any actual translations.

File: ports/libaura/portfile.cmake [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+
+vcpkg_copy_pdbs()
+
+vcpkg_fixup_pkgconfig()
+
+file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+
+configure_file("${CMAKE_CURRENT_LIST_DIR}/usage" "${CURRENT_PACKAGES_DIR}/share/${PORT}/usage" COPYONLY)
```
Comment (dg0yt) [inline ports/libaura/portfile.cmake:31]: Missing line break at EOF.

File: ports/libaura/vcpkg.json (hunk 6 above)
Comment (nlogozzo) [reply ports/libaura/vcpkg.json]: renamed to `libnick`

**Comment by nlogozzo** [comment]
@dg0yt and just wanna say thanks for working with me through this..it is my first C++ library and first time publishing to vcpkg 😆

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**